SDK/Hardware/HA4IoT.Hardware.Knx/KnxDigitalJoinEnpoint.cs

```diff
@@ -0,0 +1,51 @@
+﻿using HA4IoT.Contracts.Actuators;
+using HA4IoT.Contracts.Hardware;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace HA4IoT.Hardware.Knx
+{
+    public class KnxDigitalJoinEnpoint : IBinaryStateEndpoint
+    {
+        private readonly string _identifier;
+        private readonly KnxController _knxController;
+
+        public KnxDigitalJoinEnpoint(string identifier, KnxController knxController)
+        {
+            if (identifier == null) throw new ArgumentNullException(nameof(identifier));
+            if (knxController == null) throw new ArgumentNullException(nameof(knxController));
+
+            if (ValidationJoin(identifier))
+            {
+                _identifier = identifier;
+                _knxController = knxController;
+            }
+        }
+
+        public void TurnOn(params IHardwareParameter[] parameters)
+        {
+            _knxController.DigitalJoinOn(_identifier);
+        }
+
+        public void TurnOff(params IHardwareParameter[] parameters)
+        {
+            _knxController.DigitalJoinOff(_identifier);
+        }
+
+        public void Toggle(params IHardwareParameter[] parameters)
```

Comment:
but in my system there is a difference between turn on and toggle.
if i want to toggle a light then i send a 1 and then a 0 to my digital output. otherwise the bus is in conflict.

if i want to turn on my tv i send a 1 on the bus and that's it. No 0 or the tv will go out again